packages/common/src/components/CondaPkgList.tsx

```diff
@@ -143,6 +147,23 @@ export class CondaPkgList extends React.Component<IPkgListProps> {
       return <span>{rowData.name}</span>;
     };
 
+    const versionRender = ({ rowData }: ICellRender): JSX.Element => (
+      <a
```

Comment:
Should we handle directly here the fact that mamba will be able to render the graph but not conda?

Reply:
How do you want to do it? I'm not following you, sorry.